sys-process/runit/runit-2.1.2-r1.ebuild

```diff
@@ -71,8 +71,7 @@ dodir /etc/runit
 		#/etc/env.d/20runit
 		SVDIR="/etc/service/"
 	EOF
-	insinto /etc/env.d
-	newins "${T}"/env.d 20runit
+	newenvd "${T}"/env.d 20runit
```

Comment:
instead of writing the temporary file, use the "cat into" trick:
```
newenvd - 20runit <<- EOF
	#/etc/env.d/20runit
	SVDIR="/etc/service/"
EOF
```

Reply:
That's a great idea, thanks.